src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+$calendar-day-with: 100px;
+
+.calendar {
+  width: $calendar-day-with * 7 + 6px;
```

Comment:
The calculation for the width of the calendar uses `$calendar-day-with`, which should be corrected to `$calendar-day-width` if the intention is to use this variable for width calculations.